rdr_service/resource/generators/participant.py

```diff
@@ -606,6 +704,11 @@ def _prep_modules(self, p_id, ro_session):
                 # consent_added == True means we already know it wasn't a replayed consent
                 if consent_added or not self.is_replay(last_mod_processed, module_data, created_key='module_created'):
                     modules.append(module_data)
+                    try:
+                        activity.append(_act(row.authored, ActivityGroup.QuestionnaireModule,
+                                                 ParticipantActivity[module_name], **mod_ca))
```

Comment:
We've been moving toward letting codes be case-insensitive. Can this be updated to account for that?

Reply:
Done